### ISSUE TITLE
Add Eraser Feature

### ISSUE THREAD

**Issue by Y0ngg4n** [spacedeck/spacedeck-open#198]
Would be nice to add a Eraser to remove things on the Whiteboard.

**Comment by rds13** [comment]
You can remove a selected element via keyboard with 'backspace'.

**Comment by Y0ngg4n** [comment]
Yes but an eraser is more usefull, because you dont have to select all items you want to delete. Especially for scribbled things this is very annoying

**Comment by rds13** [comment]
You would like to be able to delete only a part of a scribble or a scribble as a whole is good enough ?

**Comment by Y0ngg4n** [comment]
a scribble as a whole is good enough :)
